BLocal/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="6.0.5" targetFramework="net35" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net35" />
```

Comment:
@KrzychuR the should it be: `net35` -> `net40`?

Reply:
@rusln I didn't test BLocal with new .NET maybe we should stay with 3.5?